src/Angor/Avalonia/Angor.Contexts.Wallet/WalletContextServices.cs

```diff
@@ -30,7 +30,9 @@ public static ServiceCollection Register(ServiceCollection services, ILogger log
         services.AddSingleton<INetworkConfiguration>(networkConfiguration);
         services.AddSingleton<INetworkService, NetworkService>();
         services.AddSingleton<INetworkStorage, NetworkStorage>();
-        services.TryAddSingleton<IIndexerService>(provider => new IndexerService(provider.GetRequiredService<INetworkConfiguration>(), provider.GetRequiredService<IHttpClientFactory>().CreateClient(), provider.GetRequiredService<INetworkService>()));
+        //TODO change the call to use the factory
```

Comment:
I'm hoping that at some point, we use the IHttpClientFactory pattern instead of injecting HttpClient directly. This can lead to potential problems. See https://learn.microsoft.com/en-us/dotnet/fundamentals/networking/http/httpclient-guidelines

Reply:
This will definitely happen but in another PR :)